Basic/Pod/QuickStart.pod

```diff
@@ -25,7 +25,7 @@ complete Object-Oriented extension to Perl (although you don't have to know
 what an object is to use it) which allows large N-dimensional data sets, such
 as large images, spectra, time series, etc to be stored  B<efficiently> and
 manipulated B<en masse>.  For example  with the PDL module we can write the
-Perl code C<$a = $b + $c>, where C<$b> and C<$c> are large datasets
+Perl code C<$x = $y + $c>, where C<$y> and C<$c> are large datasets
```

Comment:
As before, I suggest `$c`->`$z` here (twice) just so that it's more asciitecally pleasing.